data/Inputs/BatchFiles/Human_pH_SolubilityTable.json

```diff
@@ -0,0 +1,650 @@
+{
+  "Version": 74,
+  "Individuals": [
+    {
+      "Name": "I1",
+      "Seed": 1135818288,
+      "OriginData": {
+        "CalculationMethods": [
+          "Body surface area - Mosteller",
+          "SurfaceAreaPlsInt_VAR1"
+        ],
+        "Species": "Human",
+        "Population": "European_ICRP_2002",
+        "Gender": "MALE",
+        "Age": {
+          "Value": 30.0,
+          "Unit": "year(s)"
+        }
+      },
+      "Parameters": [
+        {
+          "Path": "Organism|Lumen|Caecum|pH",
+          "Value": 5.0,
+          "ValueOrigin": {
+            "Id": 28,
+            "Source": "Unknown"
```

Comment:
How can we have a source unknown coming from DB??

Reply:
Value was overwritten by user, so the Source changed from Publication to Unknown.

Reply:
ok makes sense